src/subcommand/init/shell/powershell.rs

```diff
@@ -0,0 +1,84 @@
+use super::{HookConfig, ShellConfig};
+
+use anyhow::Result;
+
+use std::borrow::Cow;
+
+pub const CONFIG: ShellConfig = ShellConfig {
+    z,
+    alias,
+    hook: HookConfig {
+        prompt: HOOK_PROMPT,
+        pwd: hook_pwd,
+    },
+};
+
+fn z(z_cmd: &str) -> String {
+    format!(
+        r#"
+function {} {{
+    function z_cd($dir) {{
+        try {{
+            Set-Location $dir -ea Stop
+            if ($env:_ZO_ECHO -eq "1") {{
+                Write-Host "$PWD"
+            }}
+        }} catch {{
```

Comment:
I didn't quite get this - why do we silently ignore exceptions here?

Reply:
I forget why I added it, but it's probably not needed now with `Test-Path` guarding the call.